---
description: AT5G17290 ; APG5, ATATG5, ATG5 ; Arabidopsis thaliana
meta_title:
title: AT5G17290
categories: [Arabidopsis thaliana]
tags: [ root,plant sperm cell ]
---

## Gene Details:
- **Gene ID:** [AT5G17290](https://www.arabidopsis.org/locus?name=AT5G17290)
- **Gene Symbol:** <u>APG5, ATATG5, ATG5</u>
- **Gene Name:** AUTOPHAGY 5, AUTOPHAGY 5
- **Description:**   autophagy protein Apg5 family;(source:Araport11)
- **TAIR Accession:** locus:2167195
- **Genome:** [Araport11_genome_release](https://www.arabidopsis.org/download/list?dir=Genes%2FAraport11_genome_release)
- **Species:** *Arabidopsis thaliana*

## Transcripts:
   -  [AT5G17290.1](https://www.arabidopsis.org/gene?name=AT5G17290.1)
## Plant Ontology Annotations:
   - [PO:0009005](https://browser.planteome.org/amigo/term/PO:0009005)&nbsp;---&nbsp;root&nbsp;---&nbsp;raíz (Spanish, exact), radices (exact, plural), radix (exact), 根 (Japanese, exact), aerial root (narrow), climbing root (narrow)
   - [PO:0000084](https://browser.planteome.org/amigo/term/PO:0000084)&nbsp;---&nbsp;plant sperm cell&nbsp;---&nbsp;célula espermática o esperma (Spanish, exact), male gamete (exact), microgamete (exact), 植物精子細胞 (Japanese, exact), sperm nucleus (related), sperm cell (broad)
## Gene Ontology:
   - [GO:0050832](https://amigo.geneontology.org/amigo/term/GO:0050832)&nbsp;---&nbsp;acts upstream of or within&nbsp;---&nbsp;defense response to fungus
   - [GO:0006501](https://amigo.geneontology.org/amigo/term/GO:0006501)&nbsp;---&nbsp;involved in&nbsp;---&nbsp;C-terminal protein lipidation
   - [GO:0006914](https://amigo.geneontology.org/amigo/term/GO:0006914)&nbsp;---&nbsp;acts upstream of or within&nbsp;---&nbsp;autophagy
   - [GO:0005737](https://amigo.geneontology.org/amigo/term/GO:0005737)&nbsp;---&nbsp;located in&nbsp;---&nbsp;cytoplasm
   - [GO:0044804](https://amigo.geneontology.org/amigo/term/GO:0044804)&nbsp;---&nbsp;involved in&nbsp;---&nbsp;nucleophagy
   - [GO:0006995](https://amigo.geneontology.org/amigo/term/GO:0006995)&nbsp;---&nbsp;involved in&nbsp;---&nbsp;cellular response to nitrogen starvation
   - [GO:0010150](https://amigo.geneontology.org/amigo/term/GO:0010150)&nbsp;---&nbsp;acts upstream of or within&nbsp;---&nbsp;leaf senescence
   - [GO:0019776](https://amigo.geneontology.org/amigo/term/GO:0019776)&nbsp;---&nbsp;contributes to&nbsp;---&nbsp;Atg8-family ligase activity
   - [GO:0000045](https://amigo.geneontology.org/amigo/term/GO:0000045)&nbsp;---&nbsp;involved in&nbsp;---&nbsp;autophagosome assembly
   - [GO:0034045](https://amigo.geneontology.org/amigo/term/GO:0034045)&nbsp;---&nbsp;is active in&nbsp;---&nbsp;phagophore assembly site membrane
   - [GO:0000422](https://amigo.geneontology.org/amigo/term/GO:0000422)&nbsp;---&nbsp;involved in&nbsp;---&nbsp;autophagy of mitochondrion
   - [GO:0034274](https://amigo.geneontology.org/amigo/term/GO:0034274)&nbsp;---&nbsp;part of&nbsp;---&nbsp;Atg12-Atg5-Atg16 complex
   - [GO:0042594](https://amigo.geneontology.org/amigo/term/GO:0042594)&nbsp;---&nbsp;acts upstream of or within&nbsp;---&nbsp;response to starvation
## Germplasm Phenotype:
   - [CS39993]()&nbsp;---&nbsp;Under a short-day photoperiod (8h light/16h dark), the atg5-1 plants grew slower resulting in smaller plants, flowered later with reduced fecundity, and showed enhanced senescence of rosette leaves than wild type; hypersensitive to either nitrogen or carbon deprivation.
   - [SAIL_129_B07]()&nbsp;---&nbsp;early senescence; accumulation of high levels of reactive oxygen species.
   - [atg5-1]()&nbsp;---&nbsp;The T-DNA insertion prevents accumulation of the ATG5 mRNA in the mutants. Absence of 50-kD and a 40-kD proteins, which correspond to the sizes of the conjugate ATG12-ATG5 and the ATG5 proteins, respectively. Homozygous atg5-1 seedlings germinated and developed normally, and flowered and set seed at the same rate as wild-type plants under a long-day photoperiod (16h light/8h dark). Under a short-day photoperiod (8h light/16h dark), the atg5-1 plants grew slower resulting in smaller plants, flowered later than wild type with reduced fecundity, and showed enhanced senescence of rosette leaves. Hypersensitive to N-deficient growth conditions (mutants presented enhanced chlorosis of the cotyledons and reduced true leaf formation). Low recovery rates (20%) after restoration of N in the media, as compared to 100% recovery rates in wildtype. Hypersensitive to carbon starvation induced by darkness (plants become severely chlorotic and flaccid faster and protein loss was higher than in wild type). In C-starved plants the protein levels of ATG7 decreased and the ATG5 increased in the mutant, whereas the protein levels of ATG8 did not drop, but the ATG8 mRNA level increased, unlike in wild type.
   - [lon2-2 atg5-5]()&nbsp;---&nbsp;IBA responsiveness to lon2-2 restored
   - [lon2-2 atg5-6]()&nbsp;---&nbsp;IBA responsiveness to lon2-2 restored
## Function-related keywords:
   - [root](/tags/root/),&nbsp;&nbsp;[plant sperm cell](/tags/plant-sperm-cell/)
## Literature:
   - [Transcriptome profiling of the response of Arabidopsis suspension culture cells  to Suc starvation.](https://www.doi.org/10.1104/pp.104.044362)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.104.044362](https://www.doi.org/10.1104/pp.104.044362);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[15310832](https://pubmed.ncbi.nlm.nih.gov/15310832/)
   - [Autophagic nutrient recycling in Arabidopsis directed by the ATG8 and ATG12  conjugation pathways.](https://www.doi.org/10.1104/pp.105.060673)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.105.060673](https://www.doi.org/10.1104/pp.105.060673);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[16040659](https://pubmed.ncbi.nlm.nih.gov/16040659/)
   - [Global analysis of Arabidopsis gene expression uncovers a complex array of  changes impacting pathogen response and cell cycle during geminivirus infection.](https://www.doi.org/10.1104/pp.108.121038)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.108.121038](https://www.doi.org/10.1104/pp.108.121038);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[18650403](https://pubmed.ncbi.nlm.nih.gov/18650403/)
   - [Transcriptome analyses show changes in gene expression to accompany pollen  germination and tube growth in Arabidopsis.](https://www.doi.org/10.1104/pp.108.126375)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.108.126375](https://www.doi.org/10.1104/pp.108.126375);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[18775970](https://pubmed.ncbi.nlm.nih.gov/18775970/)
   - [Autophagy negatively regulates cell death by controlling NPR1-dependent salicylic  acid signaling during senescence and the innate immune response in Arabidopsis.](https://www.doi.org/10.1105/tpc.109.068635)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1105/tpc.109.068635](https://www.doi.org/10.1105/tpc.109.068635);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[19773385](https://pubmed.ncbi.nlm.nih.gov/19773385/)
   - [The formation of Anthocyanic Vacuolar Inclusions in Arabidopsis thaliana and  implications for the sequestration of anthocyanin pigments.](https://www.doi.org/10.1093/mp/ssp071)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/mp/ssp071](https://www.doi.org/10.1093/mp/ssp071);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[20085894](https://pubmed.ncbi.nlm.nih.gov/20085894/)
   - [ATG8 lipidation and ATG8-mediated autophagy in Arabidopsis require ATG12  expressed from the differentially controlled ATG12A AND ATG12B loci.](https://www.doi.org/10.1111/j.1365-313X.2010.04166.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1365-313X.2010.04166.x](https://www.doi.org/10.1111/j.1365-313X.2010.04166.x);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[20136727](https://pubmed.ncbi.nlm.nih.gov/20136727/)
   - [A critical role of autophagy in plant resistance to necrotrophic fungal  pathogens.](https://www.doi.org/10.1111/j.1365-313X.2011.04553.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1365-313X.2011.04553.x](https://www.doi.org/10.1111/j.1365-313X.2011.04553.x);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[21395886](https://pubmed.ncbi.nlm.nih.gov/21395886/)
   - [Autophagy machinery controls nitrogen remobilization at the whole-plant level  under both limiting and ample nitrate conditions in Arabidopsis.](https://www.doi.org/10.1111/j.1469-8137.2012.04084.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1469-8137.2012.04084.x](https://www.doi.org/10.1111/j.1469-8137.2012.04084.x);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[22404536](https://pubmed.ncbi.nlm.nih.gov/22404536/)
   - [Physiological and metabolic consequences of autophagy deficiency for the  management of nitrogen and protein resources in Arabidopsis leaves depending on  nitrate availability.](https://www.doi.org/10.1111/nph.12307)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/nph.12307](https://www.doi.org/10.1111/nph.12307);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[23647084](https://pubmed.ncbi.nlm.nih.gov/23647084/)
   - [Stitching together the Multiple Dimensions of Autophagy Using Metabolomics and  Transcriptomics Reveals Impacts on Metabolism, Development, and Plant Responses  to the Environment in Arabidopsis.](https://www.doi.org/10.1105/tpc.114.124677)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1105/tpc.114.124677](https://www.doi.org/10.1105/tpc.114.124677);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[24808053](https://pubmed.ncbi.nlm.nih.gov/24808053/)
   - [Anthocyanin Vacuolar Inclusions Form by a Microautophagy Mechanism.](https://www.doi.org/10.1105/tpc.15.00589)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1105/tpc.15.00589](https://www.doi.org/10.1105/tpc.15.00589);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[26342015](https://pubmed.ncbi.nlm.nih.gov/26342015/)
   - [Evidence for autophagy-dependent pathways of rRNA turnover in Arabidopsis.](https://www.doi.org/10.1080/15548627.2015.1106664)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1080/15548627.2015.1106664](https://www.doi.org/10.1080/15548627.2015.1106664);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[26735434](https://pubmed.ncbi.nlm.nih.gov/26735434/)
   - [TRAF Family Proteins Regulate Autophagy Dynamics by Modulating AUTOPHAGY PROTEIN6  Stability in Arabidopsis.](https://www.doi.org/10.1105/tpc.17.00056)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1105/tpc.17.00056](https://www.doi.org/10.1105/tpc.17.00056);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[28351989](https://pubmed.ncbi.nlm.nih.gov/28351989/)
   - [Autophagy Deficiency Compromises Alternative Pathways of Respiration following  Energy Deprivation in Arabidopsis thaliana.](https://www.doi.org/10.1104/pp.16.01576)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.16.01576](https://www.doi.org/10.1104/pp.16.01576);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[28710132](https://pubmed.ncbi.nlm.nih.gov/28710132/)
   - [Increases in activity of proteasome and papain-like cysteine protease in  Arabidopsis autophagy mutants: back-up compensatory effect or cell-death  promoting effect?](https://www.doi.org/10.1093/jxb/erx482)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/jxb/erx482](https://www.doi.org/10.1093/jxb/erx482);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[29281085](https://pubmed.ncbi.nlm.nih.gov/29281085/)
   - [Autophagy contributes to sulfonylurea herbicide tolerance via GCN2-independent  regulation of amino acid homeostasis.](https://www.doi.org/10.1080/15548627.2017.1407888)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1080/15548627.2017.1407888](https://www.doi.org/10.1080/15548627.2017.1407888);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[29377765](https://pubmed.ncbi.nlm.nih.gov/29377765/)
   - [Autophagy is essential for optimal translocation of iron to seeds in Arabidopsis.](https://www.doi.org/10.1093/jxb/ery388)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/jxb/ery388](https://www.doi.org/10.1093/jxb/ery388);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[30395253](https://pubmed.ncbi.nlm.nih.gov/30395253/)
   - [The Local Phosphate Deficiency Response Activates Endoplasmic Reticulum  Stress-Dependent Autophagy.](https://www.doi.org/10.1104/pp.18.01379)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.18.01379](https://www.doi.org/10.1104/pp.18.01379);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[30510038](https://pubmed.ncbi.nlm.nih.gov/30510038/)
   - [Proteomic and lipidomic analyses of the Arabidopsis atg5 autophagy mutant reveal  major changes in endoplasmic reticulum and peroxisome metabolisms and in lipid  composition.](https://www.doi.org/10.1111/nph.15913)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/nph.15913](https://www.doi.org/10.1111/nph.15913);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[31077612](https://pubmed.ncbi.nlm.nih.gov/31077612/)
   - [Cadmium induces reactive oxygen species-dependent pexophagy in Arabidopsis  leaves.](https://www.doi.org/10.1111/pce.13597)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/pce.13597](https://www.doi.org/10.1111/pce.13597);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[31152467](https://pubmed.ncbi.nlm.nih.gov/31152467/)
   - [Autophagy-deficient Arabidopsis mutant atg5, which shows ultraviolet-B  sensitivity, cannot remove ultraviolet-B-induced fragmented mitochondria.](https://www.doi.org/10.1039/c9pp00479c)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1039/c9pp00479c](https://www.doi.org/10.1039/c9pp00479c);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33237047](https://pubmed.ncbi.nlm.nih.gov/33237047/)
   - [Salicylic acid is a key player of Arabidopsis autophagy mutant susceptibility to  the necrotrophic bacterium Dickeya dadantii.](https://www.doi.org/10.1038/s41598-021-83067-6)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/s41598-021-83067-6](https://www.doi.org/10.1038/s41598-021-83067-6);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33574453](https://pubmed.ncbi.nlm.nih.gov/33574453/)
   - [Autophagic Degradation of the 26S Proteasome Is Mediated by the Dual  ATG8/Ubiquitin Receptor RPN10 in Arabidopsis.](https://www.doi.org/10.1016/j.molcel.2021.03.026)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.molcel.2021.03.026](https://www.doi.org/10.1016/j.molcel.2021.03.026);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33961777](https://pubmed.ncbi.nlm.nih.gov/33961777/)
   - [The core autophagy machinery is not required for chloroplast singlet  oxygen-mediated cell death in the Arabidopsis thaliana plastid ferrochelatase two  mutant.](https://www.doi.org/10.1186/s12870-021-03119-x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1186/s12870-021-03119-x](https://www.doi.org/10.1186/s12870-021-03119-x);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34281507](https://pubmed.ncbi.nlm.nih.gov/34281507/)
   - [Autophagy mutants show delayed chloroplast development during de-etiolation in  carbon limiting conditions.](https://www.doi.org/10.1111/tpj.15452)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/tpj.15452](https://www.doi.org/10.1111/tpj.15452);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34365695](https://pubmed.ncbi.nlm.nih.gov/34365695/)
   - [The F-box E3 ubiquitin ligase BAF1 mediates the degradation of the  brassinosteroid-activated transcription factor BES1 through selective autophagy  in Arabidopsis.](https://www.doi.org/10.1093/plcell/koab210)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/plcell/koab210](https://www.doi.org/10.1093/plcell/koab210);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34436598](https://pubmed.ncbi.nlm.nih.gov/34436598/)
   - [γ-Aminobutyric acid plays a key role in plant acclimation to a combination of  high light and heat stress.](https://www.doi.org/10.1093/plphys/kiac010)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/plphys/kiac010](https://www.doi.org/10.1093/plphys/kiac010);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[35078231](https://pubmed.ncbi.nlm.nih.gov/35078231/)
   - [Hydrogen sulfide reduces cell death through regulating autophagy during  submergence in Arabidopsis.](https://www.doi.org/10.1007/s00299-022-02872-z)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00299-022-02872-z](https://www.doi.org/10.1007/s00299-022-02872-z);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[35507055](https://pubmed.ncbi.nlm.nih.gov/35507055/)
   - [Type one protein phosphatase regulates fixed-carbon starvation-induced autophagy  in Arabidopsis.](https://www.doi.org/10.1093/plcell/koac251)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/plcell/koac251](https://www.doi.org/10.1093/plcell/koac251);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[35961047](https://pubmed.ncbi.nlm.nih.gov/35961047/)
   - [TraB family proteins are components of ER-mitochondrial contact sites and  regulate ER-mitochondrial interactions and mitophagy.](https://www.doi.org/10.1038/s41467-022-33402-w)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/s41467-022-33402-w](https://www.doi.org/10.1038/s41467-022-33402-w);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[36163196](https://pubmed.ncbi.nlm.nih.gov/36163196/)
   - [Ufmylation reconciles salt stress-induced unfolded protein responses via ER-phagy  in Arabidopsis.](https://www.doi.org/10.1073/pnas.2208351120)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1073/pnas.2208351120](https://www.doi.org/10.1073/pnas.2208351120);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[36696447](https://pubmed.ncbi.nlm.nih.gov/36696447/)
   - [The phosphatidylinositol 3-phosphate effector FYVE3 regulates FYVE2-dependent  autophagy in Arabidopsis thaliana.](https://www.doi.org/10.3389/fpls.2023.1160162)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3389/fpls.2023.1160162](https://www.doi.org/10.3389/fpls.2023.1160162);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[37008475](https://pubmed.ncbi.nlm.nih.gov/37008475/)
   - [FRIENDLY is required for efficient dark-induced mitophagy and controlled  senescence in Arabidopsis.](https://www.doi.org/10.1016/j.freeradbiomed.2023.04.007)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.freeradbiomed.2023.04.007](https://www.doi.org/10.1016/j.freeradbiomed.2023.04.007);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[37085125](https://pubmed.ncbi.nlm.nih.gov/37085125/)
   - [Autophagic nutrient recycling in Arabidopsis directed by the ATG8 and ATG12  conjugation pathways.](https://www.doi.org/10.1104/pp.105.060673)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.105.060673](https://www.doi.org/10.1104/pp.105.060673);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[16040659](https://pubmed.ncbi.nlm.nih.gov/16040659/)
## Sequence:
{{< accordion "cDNA Sequence" >}}
- \>AT5G17290.1
CGTACGCCTTATTAATACCACATAAAGACGCAAGAGAAGATGACGTCATACAGAAACAGCGTCGTTTTGAAAGCTGAGTTTTTTTTGTGAAGCGAGCGAGCGTGAGAAGGTGTGACGGGGAAGAGAATGGCGAAGGAAGCGGTCAAGTATGTATGGGAAGGAGCAATTCCTCTGCAGATTCATCTCCACAAATCCGACGTCGCTTCTCACCCTGCTCCTCCTCCTGCTCTTGTGTTAGCACCAAGAATAGGATATTTGCCTCTGTTGATTCCTCTTATAAAGCCTTATTTCAAGGATTCACTTCCTCCTGGTGAAGATTCAATTTGGTTTGATTACAAAGGATTTCCTCTAAAATGGTATATACCAACAGGTGTTCTTTTCGATCTCCTTTGTGCAGAACCCGAAAGACCATGGAATCTCACGATACACTTTAGAGGATATCCTTGCAACATACTGATACCATGTGAAGGAGAAGATTCTGTAAAATGGAACTTTGTTAATTCTTTGAAAGAGGCACAATATATCATCAATGGAAATTGCAAGAATGTTATGAACATGTCTCAGAGTGATCAAGAGGATCTATGGACCTCTGTCATGAACGGTGATCTTGATGCCTATACAAGATTATCACCCAAGCTTAAAATGGGAACAGTCGAAGATGAGTTTTCAAGGAAAACAAGTTTGTCATCTCCACAATCTCAACAAGTTGTGCCTGAGACGGAGGTGGCTGGACAAGTTAAGACAGCAAGAATTCCTGTTCGGTTGTATGTTCGAAGTCTAAATAAAGATTTCGAGAATCTTGAAGATGTACCGGAGATCGATACCTGGGATGACATCTCGTACCTTAATCGCCCTGTTGAGTTCCTCAAAGAAGAAGGGAAATGCTTTACGTTACGTGACGCCATTAAAAGTCTCCTCCCTGAGTTTATGGGAGACAGAGCGCAAACGAGTGGTGAAGAAAGAAGCATAGATGATACAGAAGAAGCAGATGGGTCGAGGGAGATGGGTGAAATCAAATTGGTAAGGATACAAGGGATAGAAATGAAGCTAGAGATACCGTTTTCGTGGGTGGTAAATAACTTGATGAACCCAGAATTCTATCTCCATATCTCTGTCCTTGTGAAAGCTCCTCAAAGGTGAAGTGTAAGGTTCTCTGCAGTTACAATCCATCTGTGAATTGAATCAAATTGCTTTCTCGTTCCATCTTACAAATCCGAAAGAATCAATGATTTGTTGTATACAGCTACTTCTTCTATTTCTGATAGAAGCAAGAACACAGAATACAGAAAAGAAAGTAGAAATATTTTGATTGACTCGTTAGCTCTTTTACATTTGTTACTTGACGGTTTCCTCTCTGAGCAAATTTGATTGAAATCAATTTATGGTCGATATAATAATCACTACTTTCTTCTATTCACAAGTAAATCAGCTCAAGTTCATAGAGAGATCCAAAATGTTGTTCAACAAAAGCACTGTATCATCAAATCGACAAAGACAAATGTTTAATTTCTCCATTAACAATGCGAGGAAAAATCAGTCAAATTGATTTAGTTTCTCCAGAAAGCTGCTTGTTGTAATCTTCGAGCTCATCGTAATAAACATCCCAAACGCATCGGACGCAACCGCTACCGCAACAATCGCCAGGCTCTGGTTTCTCTGGCGGTGGAGGAACCGAAACTCCCACCTCCTTCTTATCTTCTGTCTCCTTCTTGTTTGTCTCCTCCTTGGAAGCTTCTACAAGATTCTTGCTCTCGGTCGTCGCCATGGTCACCGGAGATTTGAGCTGTTGTTCTGGAGATACGGAAGTGATTGAATAATTGGAGAAAGATCGTTTGAGAGGGAGATGTCGCGTCAAACCAAATCTCATGCTCAAAAGCACATCGTGAAGGCTAGAACCCGGTGATGTAACGATCGAGATTCGAGGAAGAAGAGACACAACAACCATACGCACAGATCAAAAAGATTTGTTCTCTTCTTTTTTTTTCTATTTCCTTTTTTCGTGTCAGAATTGATTTCTTTATGACCTGTCATCCTATAGTTGGCAAAAAAAGCCGCCAATCATAAACGGCACGCATATTTATCTTCTAAACGTTAAAACGGCACGACATCTTTTTTGAACACAAGAACCAACAACTTCTTGATCTTCAGCTGCGCCATGATTTTGA
{{< /accordion >}}
{{< accordion "CDS Sequence" >}}

{{< /accordion >}}
{{< accordion "Protein Sequence" >}}

{{< /accordion >}}
